### PR TITLE
docs: update session_tracker reference

### PR DESCRIPTION
**Pull request by stevenGravy**
verbiage sounded like only applied to ssh.

### CODE REVIEW

File: docs/pages/reference/access-controls/roles.mdx [hunk 1]
```diff
@@ -628,7 +628,7 @@ within this guide:
 | -------------------------- | ------------------------------------------------- |
 | `user.spec.roles`          | The list of roles assigned to a user              |
 | `session.participants`     | The list of participants from a session recording |
-| `session_tracker.participants` | The list of participants from an SSH session      |
+| `session_tracker.participants` | The list of participants from an SSH or Kubernetes session      |
```
Comment (zmb3) [inline docs/pages/reference/access-controls/roles.mdx:631]:
```suggestion
| `session_tracker.participants` | The list of participants from the session      |
```
Session trackers exist for all types of sessions.

File: docs/pages/reference/access-controls/roles.mdx [hunk 2]
```diff
@@ -405,7 +405,7 @@ spec:
     rules:
     # ... and then limit that access via a deny rule.
     # Deny rules take precedence over allow rules, so the resulting role allows
-    # users to create SSH sessions but to only view their own sessions.
+    # users to create sessions including SSH ones but to only view their own sessions.
```
Comment (zmb3) [inline docs/pages/reference/access-controls/roles.mdx:408]: Let's simplify the language here.

```yaml
# Deny rules take precedence over allow rules, so the resulting role
# only allows user to see their own active sessions.
```

Comment (stevenGravy) [reply docs/pages/reference/access-controls/roles.mdx]: thanks @zmb3 updated

**Review by zmb3 — APPROVED** [review]

**Review by scottpgallagher — APPROVED** [review]
Approved.

**Pull request merged**